pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1348,8 +1348,9 @@ type ObjectStoreSpec struct {
 
 // ObjectHealthCheckSpec represents the health check of an object store
 type ObjectHealthCheckSpec struct {
-	// +optional
-	LivenessProbe *ProbeSpec `json:"livenessProbe,omitempty"`
+	// livenessProbe field is no longer used
+	// +kubebuilder:pruning:PreserveUnknownFields
```

Comment:
This preserve tag applies to all the health check and not just the readiness probe, right?

Reply:
Right. The other option is to still include the livenessProbe spec item and ignore it, but that seems like it could confuse any new users. Maybe it would be better thought? I'm open to thoughts